src/app/pages/payment/payment.component.scss

```diff
@@ -0,0 +1,54 @@
+.ng-invalid {
```

Comment:
Hey @Victoradukwu, I notice that when a user has validation errors, the fields with problems highlight red but no descriptive messages are shown, it can be better if we can guide the user as much as possible such that they know how to correctly fill the form.